### PR TITLE
[Docs] Minor documentation updates during walkthrough

### PR DESCRIPTION
**Pull request by Olshansk**
Updated some documentation while doing an end-to-end walkthrough.

### CODE REVIEW

File: cmd/gateway_server/internal/transform/qos_node.go [hunk 1]
```diff
@@ -1,9 +1,10 @@
 package transform
 
 import (
+	"math"
```
Comment (commoddity) [inline cmd/gateway_server/internal/transform/qos_node.go:4]: why is this import added?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+version: "3"
```
Comment (commoddity) [inline docker-compose.yml]: version is no longer needed for docker-compose.yml files

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      - bridged_network
+    depends_on:
+      - postgres
+    restart: on-failure
+    logging:
+      driver: json-file
+      options:
+        max-size: 10m
+
+  # this postgres database is only to be used for testing. It should not be used in production systems
```
Comment (commoddity) [inline docker-compose.yml]: I kind of like keeping the docker-compose.yml .gitignore'd so that people can choose their own local configuration without having it automatically added to the repo.

File: docs/api-endpoints.md [hunk 4]
```diff
@@ -1,16 +1,76 @@
...
-| `/poktapps`          | GET         | A list of all the available app stakes                                                                                                             | `x-api-key` | N/A                                      |
-| `/poktapps`          | POST        | Adds an existing app stake to the appstake database (not recommended due to security)                                                              | `x-api-key` | `private_key` - private key of app stake |
-| `/poktapps/{app_id}` | DELETE      | Removes an existing app stake from the appstake database (not recommended due to security)                                                         | `x-api-key` | `app_id` -  id of the appstake           |
-| `/qosnodes`          | GET         | A list of nodes and public QoS state such as healthiness and last known error. This can be used to expose to node operators to improve visibility. | `x-api-key` | N/A                                      |
\ No newline at end of file
+_TODO_IMPROVE: Move this to Swagger in the future if our API endpoints become more complex._
```
Comment (commoddity) [inline docs/api-endpoints.md:9]: Agreed. We can also add an OpenAPI spec to this repo, which I think would be useful if we are going to keep the "admin" endpoints (arguable if they are needed TBH, although can be useful for debugging purposes).

File: .gitignore [hunk 5]
```diff
@@ -50,7 +50,13 @@
 .env
 
 ## docker compose file
-docker-compose.yml
+# docker-compose.yml
```
Comment (commoddity) [inline .gitignore]: See my comment below about keeping this .gitignore'd. I prefer having it be excluded from the repo so that local config can be changed without committing.

File: .env.sample [hunk 6]
```diff
@@ -1,11 +1,19 @@
...
-DB_CONNECTION_URL=postgres://myuser:mypassword@postgres:5432/postgres?sslmode=disable
-API_KEY=
 ALTRUIST_REQUEST_TIMEOUT=10s
-EMIT_SERVICE_URL_PROM_METRICS=false
-CHAIN_NETWORK=morse_mainnet
\ No newline at end of file
+API_KEY=
+
+# App Stake Management
+DB_CONNECTION_URL=postgres://myuser:mypassword@postgres:5432/postgres?sslmode=disable
```
Comment (commoddity) [inline .env.sample]: The default port for the DB in their example is 5433 so I think we should change either the example docker compose or example .env to match.

**Review by commoddity — COMMENTED** [review]
Looks good over all, especially the changes to the docs but a few comments to discuss.

**Comment by Olshansk** [comment]
@nodiesBlade PTAL when you have a chance.

**Review by nodiesBlade — APPROVED** [review]
skimmed through the changes, looks good. Minor formatting and comments
